### PR TITLE
Fixed component opportunitySideCard not listed under Organisms

### LINKED ISSUE #928: component opportunitySideCard not listed under Organisms
**Describe the bug**
Currently, the component `opportunitySideCard` is not titled under `Organisms` at `/interface` route, though the component is already implemented. 

**To Reproduce**
1. Go to `/interface` route.
2. Under `Organisms`, you won't find any component titled `Opportunity Side Card`

**Expected behaviour**
There should be a component under `Organisms` that is titled `Opportunity Side Card`.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

### PR DESCRIPTION
**Pull request by Sasha411**
Fixes #928 

Added opportunitySideCard under organism.

Issue928

### CODE REVIEW

File: src/ignitus-UserInterfaceBook/InterfaceSideNavigation/constants.ts [hunk 1]
```diff
@@ -209,6 +209,10 @@ export const allEdges: Edges[] = [
             title: 'Message Card',
             route: '/interface/messageCard',
           },
+          {
+            title: 'OpportunitySideCard',
+            route: '/interface/opportunitySideCard',
```
Comment (Dishebh) [inline src/ignitus-UserInterfaceBook/InterfaceSideNavigation/constants.ts:214]: Change the title to `Opportunity Side Card`

**Comment by Dishebh** [comment]
Mention the issue you're fixing in the PR description, and change the PR title so that it displays some relevant info (take help from other open PRs) :)

**Comment by Dishebh** [comment]
Also, it would be great if you can add a `Heading` in this component as well (like the one present in `Message Card` component)

File: src/ignitus-UserInterfaceBook/InterfaceSideNavigation/constants.ts (hunk 1 above)
Comment (Sasha411) [reply src/ignitus-UserInterfaceBook/InterfaceSideNavigation/constants.ts:214]: Done.

**Comment by Dishebh** [comment]
Fix eslint errors

File: src/ignitus-UserInterfaceBook/Components/Organisms/interfaceOpportunitySideCard/Components/index.tsx [hunk 2]
```diff
@@ -7,6 +7,6 @@ export const interfaceOpportunitySideCard = () => (
   <Interface>
     <Heading2>Opportunity Side Card</Heading2>
     <hr />
-  <InfoCard location="Australia" commitment="Full-time" salary="24" />
+    <InfoCard location="Australia" commitment="Full-time" salary="24" />
```
Comment (Sasha411) [inline src/ignitus-UserInterfaceBook/Components/Organisms/interfaceOpportunitySideCard/Components/index.tsx:10]: I dont know why fixing lint errors modified this but I am committing to whatever fix it did.

**Comment by Dishebh** [comment]
> Mention the issue you're fixing in the PR description, and change the PR title so that it displays some relevant info (take help from other open PRs) :)

@Sasha411, kindly do this as well

**Comment by Sasha411** [comment]
> > Mention the issue you're fixing in the PR description, and change the PR title so that it displays some relevant info (take help from other open PRs) :)
> 
> @Sasha411, kindly do this as well

Done. Sorry forgot about doing this.

**Review by Dishebh — APPROVED** [review]

**Pull request merged**